<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planner</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/apple.PNG</normaloff>../resources/apple.PNG</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="calc">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QChartView" name="graphicsView">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255,255,255,70);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QChartView" name="graphicsView_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255,255,255,70);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <widget class="table" name="tableView">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,70);</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="table" name="tableView_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,70);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="task">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0" colspan="4">
         <widget class="QToolBox" name="toolBox">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,70);</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>564</width>
             <height>587</height>
            </rect>
           </property>
           <attribute name="label">
            <string>计划</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="1">
             <widget class="QProgressBar" name="progressBar">
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255,255,255,70);</string>
              </property>
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>任务进度：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="list" name="listView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>564</width>
             <height>587</height>
            </rect>
           </property>
           <attribute name="label">
            <string>已完成</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="list" name="listView_2">
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255,255,255,70);</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>564</width>
             <height>587</height>
            </rect>
           </property>
           <attribute name="label">
            <string>添加</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="3" column="2">
                   <widget class="QDateTimeEdit" name="dateTimeEdit_2">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(255,255,255,70);</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="2">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>开始时间：</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>任务：</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0" colspan="2">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>结束时间：</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1" colspan="2">
                   <widget class="QLineEdit" name="lineEdit"/>
                  </item>
                  <item row="1" column="1" colspan="2">
                   <widget class="QComboBox" name="comboBox">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(255,255,255,70);</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>优先级：</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="2">
                   <widget class="QDateTimeEdit" name="dateTimeEdit">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(255,255,255,70);</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="1">
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>详细描述：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QTextEdit" name="textEdit">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(255,255,255,70);</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="ok">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(255,255,255,70);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resources/37CF57182DDEB6469851836D20F00638.png</normaloff>../resources/37CF57182DDEB6469851836D20F00638.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>80</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>29</width>
                    <height>17</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="cansle">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(255,255,255,70);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resources/EC5CB6F3435B6BC9B80A3D2CBC3993B4.png</normaloff>../resources/EC5CB6F3435B6BC9B80A3D2CBC3993B4.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>80</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>57</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>219</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>218</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;隶书&quot;;</string>
          </property>
          <property name="text">
           <string>学习任务</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="actiona"/>
    <addaction name="actionb"/>
    <addaction name="actioncalendar"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiona">
   <property name="text">
    <string>task</string>
   </property>
  </action>
  <action name="actionb">
   <property name="text">
    <string>calc</string>
   </property>
  </action>
  <action name="actioncalendar">
   <property name="text">
    <string>calendar</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>table</class>
   <extends>QTableView</extends>
   <header>table.h</header>
  </customwidget>
  <customwidget>
   <class>list</class>
   <extends>QListView</extends>
   <header>list.h</header>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
